cs6232-g4/Controller/MembersController.cs

```diff
@@ -11,11 +11,11 @@ namespace Members.Controller
 {
     public class MembersController
     {
-        private readonly MembersDAL _memberDAL;
+        private readonly LoginDAL _memberDAL;
```

Comment:
There's already a Login controller. why using MemberController to access LoginDAL ?